conversation/frontend/src/features/message-list/message-preview/components/RecipientListPreview.tsx

```diff
@@ -17,7 +21,7 @@ export function RecipientListPreview({ message }: RecipientListPreviewProps) {
   };
   return (
     <MessageRecipientList
-      head={t('at')}
+      head={hasPrefix ? t('at') : null}
```

Comment:
```suggestion
      head={head}
```
ou
`{...otherProps}`